registration-processor/post-processor/registration-processor-registration-transaction-service/src/main/java/io/mosip/registration/processor/transaction/api/controller/handler/RegistrationTransactionExceptionHandler.java

```diff
@@ -116,11 +119,15 @@ private ResponseEntity<RegTransactionResponseDTO> buildRegTransactionExceptionRe
 		response.setResponsetime(DateUtils.getUTCCurrentDateTimeString(env.getProperty(DATETIME_PATTERN)));
 		response.setVersion(env.getProperty(REG_TRANSACTION_APPLICATION_VERSION));
 		response.setResponse(null);
-		Gson gson = new GsonBuilder().serializeNulls().create();
+		objMp.setSerializationInclusion(JsonInclude.Include.NON_NULL);

```

Comment:
here too. why did u add JsonInclude.Include.NON_NULL? was it a  requirement? it may break caller if not handled properly